.drone.yml

```diff
@@ -132,6 +132,9 @@ steps:
 
 - name: trigger
   image: plugins/downstream
+  # This step fails when 'packaging' hasn't a branch with the
```

Comment:
Why is this ignored?  Packaging will never have a branch with the same name except on `master`.

Reply:
For instance, I'm not going to have a packages repository for some assurio-snap branch. And I'll not create appropriate packaging branch. Why build have to be failed in this case?
If we need some "feature build" packages, then I'll just branch master in the packaging with the same branch name.

Reply:
Oh.  I didn't realize that's how your packaging scripts work.  That's not acceptable.  Please create a git issue in the `packaging` repo about the problem that there must be a matching branch in `packaging` for every branch build in every other repo.

Once you do that I'll approve this.

Reply:
https://github.com/assuriosw/packaging/issues/5